addons/binding/org.openhab.binding.irtrans/src/main/java/org/openhab/binding/irtrans/handler/EthernetBridgeHandler.java

```diff
@@ -0,0 +1,1078 @@
+                    }
+                }
+            } else {
+                try {
+                    logger.warn("Did not receive an answer from the IRtrans transceiver '{}' - Parsing is skipped",
+                            socketChannel.getRemoteAddress());
+                    disconnect(socketChannel);
+                } catch (IOException e1) {
+                    logger.debug("An exception occurred while getting a remote address: '{}'", e1.getMessage());
+                }
+            }
+
+            int numberOfRemotes = 0;
+            int numberOfRemotesProcessed = 0;
+            int numberOfRemotesInBatch = 0;
+            String[] remoteList = getRemoteList(0);
+
+            if (remoteList.length > 0) {
+                logger.info("The IRtrans device for '{}' supports '{}' remotes", getThing().getUID(), remoteList[1]);
+                numberOfRemotes = Integer.valueOf(remoteList[1]);
+                numberOfRemotesInBatch = Integer.valueOf(remoteList[2]);
+            }
+
+            while (numberOfRemotesProcessed < numberOfRemotes) {
+                for (int i = 1; i <= numberOfRemotesInBatch; i++) {
+                    String remote = remoteList[2 + i];
+
+                    // get remote commands
+                    String[] commands = getRemoteCommands(remote, 0);
+                    String resultString = new String();
+                    int numberOfCommands = 0;
+                    int numberOfCommandsInBatch = 0;
+                    int numberOfCommandsProcessed = 0;
+
+                    if (commands.length > 0) {
+                        numberOfCommands = Integer.valueOf(commands[1]);
+                        numberOfCommandsInBatch = Integer.valueOf(commands[2]);
+                        numberOfCommandsProcessed = 0;
+                    }
+
+                    while (numberOfCommandsProcessed < numberOfCommands) {
+                        for (int j = 1; j <= numberOfCommandsInBatch; j++) {
+                            String command = commands[2 + j];
+                            resultString = resultString + command;
+                            numberOfCommandsProcessed++;
+                            if (numberOfCommandsProcessed < numberOfCommands) {
+                                resultString = resultString + ", ";
+                            }
+                        }
+
+                        if (numberOfCommandsProcessed < numberOfCommands) {
+                            commands = getRemoteCommands(remote, numberOfCommandsProcessed);
+                            if (commands.length == 0) {
+                                break;
+                            }
+                            numberOfCommandsInBatch = Integer.valueOf(commands[2]);
+                        } else {
+                            numberOfCommandsInBatch = 0;
+                        }
+
+                    }
+
+                    logger.info("The remote '{}' on '{}' supports '{}' commands: {}",
```

Comment:
Please change the logging level to `debug`. `trace` can be used for even more details, where necessary.